dags/taar_amodump.py

```diff
@@ -55,27 +81,37 @@
     instance_count=1,
     owner="vng@mozilla.com",
     email=["mlopatka@mozilla.com", "vng@mozilla.com"],
-    env=mozetl_envvar("taar_update_whitelist",
-                      {"date": "{{ ds_nodash }}"},
-                      {'MOZETL_SUBMISSION_METHOD': 'spark'}),
+    env=mozetl_envvar(
+        "taar_update_whitelist",
+        {"date": "{{ ds_nodash }}"},
+        {"MOZETL_SUBMISSION_METHOD": "spark"},
+    ),
     uri="https://raw.githubusercontent.com/mozilla/python_mozetl/master/bin/mozetl-submit.sh",
     output_visibility="private",
-    dag=dag
+    dag=dag,
 )
 
-taar_lite = EMRSparkOperator(
+
+# Spark job reads gcs json and writes gcs parquet
+taar_lite = SubDagOperator(
     task_id="taar_lite",
-    job_name="Generate GUID coinstallation JSON for TAAR",
-    instance_count=5,
-    execution_timeout=timedelta(hours=4),
-    owner="mlopatka@mozilla.com",
-    email=["mlopatka@mozilla.com", "vng@mozilla.com"],
-    env=mozetl_envvar("taar_lite",
-                      {"date": "{{ ds_nodash }}"},
-                      {'MOZETL_SUBMISSION_METHOD': 'spark'}),
-    uri="https://raw.githubusercontent.com/mozilla/python_mozetl/master/bin/mozetl-submit.sh",
-    output_visibility="private",
-    dag=dag
+    dag=dag,
+    subdag=moz_dataproc_pyspark_runner(
+        parent_dag_name=dag.dag_id,
+        default_args=default_args,
+        cluster_name=cluster_name,
+        job_name="Generate GUID coinstallation JSON for TAAR",
+        # TODO - CI/CD for gcs and repo sync for artifact bucket jobs and gcp bootstrap folders
+        # TODO: the currentcode lives at - need to get it into GCS
+        # https://github.com/mozilla/python_mozetl/blob/master/mozetl/taar/taar_lite_guidguid.py
+        python_driver_code="gs://moz-fx-data-prod-airflow-dataproc-artifacts/jobs/socorro_import_crash_data.py",
```

Comment:
this path should point to the same gcs bucket, but then a file in telemetry-airflow/jobs  (which will be sync'd to that bucket on master commits)

Reply:
For testing purposes, i can drop the script there for you manually